handlers/providers/rackspace/rackspace.go

```diff
@@ -0,0 +1,29 @@
+package rackspace
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/rancherio/go-machine-service/handlers/providers"
+	"github.com/rancherio/go-rancher/client"
+)
+
+func init() {
+	rackspaceHandler := &RackspaceHandler{}
+	if err := providers.RegisterProvider("rackspace", rackspaceHandler); err != nil {
+		log.Fatal("could not register rackspace provider")
+	}
+}
+
+type RackspaceHandler struct {
+}
+
+func (*RackspaceHandler) HandleCreate(machine *client.Machine, machineDir string) error {
+	return nil
+}
+
+func (*RackspaceHandler) HandleError(msg string) string {
+	prettyMsg := msg
+	if msg == "Expected HTTP response code [200 203] when accessing [POST https://identity.api.rackspacecloud.com/v2.0/tokens], but got 401 instead" {
```

Comment:
I don't know of any other solution other than look for specific codes rather than do an exact match of the error string itself. It's highly unlikely providers, or at least established ones, will change these messages often.